.github/workflows/cancel-old-reviews.yaml

```diff
@@ -0,0 +1,17 @@
+on:
+    schedule:
+        -   cron: '0 2 * * *'
+name: Cancel reviews older than seven days
+jobs:
+    rebuild-master-images:
```

Comment:
shouldn't be this job named differently?

Reply:
Good catch!